authortechnical.Rmd

```diff
@@ -34,19 +31,44 @@ _In Hugo speak, we'd say your post is a [leaf bundle](https://gohugo.io/content-
 
 ## Start the post from a template {#templates}
 
-Prepare your post in your favorite editor in R Markdown **or** Markdown by starting with templates we provide in this book or [from the roblog package](#roblogtemplates).
+Prepare your post in your favorite editor in R Markdown **or** Markdown by starting with templates we provide in this book or using blogdown's handy New Post RStudio addin (recommended approach).
```

Comment:
I'd be strongly in favour of making this the recommended approach because as @steffilazerte noted, handling paths is a bit confusing. With the blogdown addin, there's more setup (installing two packages, installing Hugo) but then less mistakes possible.